AnkiDroid/build.gradle

```
@@ -154,6 +154,7 @@ android {
         preDexLibraries = preDexEnabled && !ciBuild
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true // Enabling desugaring.
```

As mentioned in my previous review comment, code comments in general are good but: *they must add information* - a comment that does not add information is just more to read + think about that in the end performs no service.

I believe we can all agree that if we are setting a variable called `coreLibraryDesugaringEnabled` to `true` it is obvious we are `// Enabling desugaring.` yes? So the comment adds no value and thus takes time and attention for no purpose.

```suggestion
        coreLibraryDesugaringEnabled true
```